run.py

```diff
@@ -1,34 +1,41 @@
+#!/usr/bin/env python3
+import subprocess
 import time, os, shutil, json
+from subprocess import run
+
+
+def get_battery_percentage():
+    battery_percentage_command = "cat $(find /sys/class/power_supply/ -name 'BAT*' | head -n 1)/capacity"
+    result = run(
+        battery_percentage_command,
+        shell=True,
+        stdout=subprocess.PIPE,
+        universal_newlines=True)
+    return result.stdout.strip()
+
 
 def main():
-    bgImage = ""
+    background_image = ""
     while True:
         # Runs Script that gets current Battery Charge
-        os.system("./battery.sh")
-        time.sleep(1)
-        # Reads The Charge. Note: Looking for a better way to do this
-        with open('data.json', 'r') as f:
-            data = json.load(f)
-            val = int(data["Charge"])
-        a = 100
```

Comment:
no need to start at a = 100 when the highest value to check is the battery percentage